rules/mocha.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+  rules: {
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-skipped-tests': 'off',
+    'mocha/no-pending-tests': 'error',
+    'mocha/handle-done-callback': 'error',
```

Comment:
This won't trigger an error if we don't define the argument (because we're returning a promise), right?